SPECS/grpc/grpc.spec

```diff
@@ -67,12 +88,29 @@ cmake ../.. -DgRPC_INSTALL=ON                \
    -DgRPC_RE2_PROVIDER:STRING='package'      \
    -DgRPC_SSL_PROVIDER:STRING='package'      \
    -DgRPC_ZLIB_PROVIDER:STRING='package'
-%make_build
+%cmake_build
+popd
+#python
+export GRPC_PYTHON_BUILD_WITH_CYTHON=True
+export GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=True
+export GRPC_PYTHON_BUILD_SYSTEM_ZLIB=True
+export GRPC_PYTHON_BUILD_SYSTEM_CARES=True
+export GRPC_PYTHON_BUILD_SYSTEM_RE2=True
+export GRPC_PYTHON_BUILD_SYSTEM_ABSL=True
+%py3_build
+
 
 %install
-cd cmake/build
-%make_install
+pushd cmake/build
+%cmake_install
 find %{buildroot} -name '*.cmake' -delete
+popd
+#python
+pushd '%{buildroot}'
+PYROOT="${PWD}"
+popd
```

Comment:
I think I don't understand this part. Why do we need to go to `%{buildroot}` just to get the directory path? And then we set `PYROOT` but since it's not exported or set through `PYTROOT=abc %py3_install`, it's not being utilized by anything anyway. Could you check, if we actually need this part?
```suggestion
```

Reply:
removed..